manage.py

```diff
@@ -93,7 +101,7 @@ def seed(include_interventions=False, keep_unmentioned=False):
     SitePersistence().import_(include_interventions, keep_unmentioned)
 
 
-@manager.command
+@app.cli.command()
 def export_site():
```

Comment:
It might - but we never use this anymore, as too many hand edits have since taken place on the site persistence file, and the generated order of JSON attributes isn't consistent.  That was by design initially, and will only function if `PERSISTENCE_FILE` is pointing to a filesystem (not URL) location.